### ISSUE TITLE
Debug (Not available) on Performance Analyzer query

### ISSUE THREAD

**Issue by Mike-Honey** [TabularEditor/TabularEditor3#846]
1st time using this tool so possible newbie error. 
I connected to my Power BI Desktop session using Model From DB / Local Instance. Not sure I followed the right steps as the terminology was confusing.
In PBI Desktop I used the Performance Analyzer in Power BI to capture a slow query for a matrix visual and pasted it into TE, hit execute and it ran. A bit stumped then, but eventually realised there were 2 results windows, and the data for my matrix was on the 2nd one.

I right-clicked in the results pane, but for every cell it displays 'Debug (not available)'. The measures used are all explicit.

I'm stuck.

**Comment by edhans** [comment]
Yes, DAX queries can generate multiple results, and the one you are _usually_ interested in is the last. I've seen a visual generate 3 results to get the data it needs to render.

I've also run into issues with Debug not being available, and it isn't always clear to me why that happens. Try to remove all measures from the visual except the one you are interested in and see if debug becomes available. 

Just curious, do you have any Calculation Groups acting on this visual?

**Comment by mlonsk** [comment]
Hi @Mike-Honey 
Glad to hear you are getting into Tabular Editor.
With regard to the lack of debugging options, this is most likely due to some of the limitations that we currently have with the debugger: [DAX Debugger Limitations](https://docs.tabulareditor.com/te3/features/dax-debugger.html#limitations-and-known-issues)

Does your DAX query from the performance analyser contain any of those?

When I use the performance analyser queries, I prefer to strip as much away as possible to only have the core query left and then debug that.
